tests/gaggle.rs

```diff
@@ -36,11 +36,13 @@ fn test_gaggle() {
         master_configuration.manager = true;
         master_configuration.expect_workers = 1;
         master_configuration.run_time = "3".to_string();
-        crate::GooseAttack::initialize_with_config(master_configuration)
+        let _stats = crate::GooseAttack::initialize_with_config(master_configuration)
```

Comment:
Technically it's a goose attack and not stats ...

Reply:
fixed